src/internal/translations/en-US.json

```diff
@@ -399,6 +399,46 @@
         "comment": "",
         "limit": 0
     },
+    "properties.axis.name": {
+        "value": "name",
+        "comment": "",
```

Comment:
Please add guide how to translate the text.
More info [here](https://github.com/gooddata/gdc-dashboards/pull/2268#discussion_r323172893)

Reply:
Fixed

Reply:
I didn't see the guide "Translate as imperative" or something like that.
[Example](https://github.com/gooddata/gdc-dashboards/pull/2322/files#diff-186afe801e617466704d7ead1b2a37bdR1914) 

Please add the guide for all of the new texts